ftl/common/cache.py

```diff
@@ -99,3 +100,58 @@ def Store(self, base_image, namespace, checksum, value):
                 threads=self._threads,
                 mount=self._mount) as session:
             session.upload(value)
+
+
+class MockHybridRegistry(Base):
```

Comment:
We already have this in test_util, any reason to have it here as well?

Reply:
no, removing